global-raw-tokens/src/main/java/com/orange/ouds/tokens/OpacityRawTokens.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.tokens
+
+object OpacityRawTokens {
+    const val opacity0 = 0f
+    const val opacity100 = 0.04f
+    const val opacity200 = 0.08f
+    const val opacity300 = 0.16f
+    const val opacity400 = 0.24f
+    const val opacity500 = 0.32f
+    const val opacity600 = 0.48f
+    const val opacity700 = 0.64f
+    const val opacity800 = 0.88f
```

Comment:
I wonder if there is an error in Figma, because this value should be `0.82f` if we keep the same logic.